### ISSUE TITLE
TypeError: Cannot read property 'content' of undefined

### ISSUE THREAD

**Issue by andrejs-uibo** [cascornelissen/svg-spritemap-webpack-plugin#146]
When trying to build assets, I get this error:
```
node_modules/svg-spritemap-webpack-plugin/lib/index.js:209
                        this.stylesType = this.getStylesType(this.styles.content, stylesOptions.filename);
                                                                         ^
TypeError: Cannot read property 'content' of undefined
```
Here is a part of webpack config how I use this plugin:
```
plugins: [
    new SVGSpritemapPlugin("frontend/web/images/features/*.svg", {
      output: {
        filename: "features.svg",
      },
      styles: {
        keepAttributes: true,
        filename: "features.css",
      },
    }),
...
```
This is on webpack v5.16.0
Plugin v3.8.3

**Comment by andrejs-uibo** [comment]
This could be related to https://github.com/cascornelissen/svg-spritemap-webpack-plugin/issues/143

**Comment by cascornelissen** [comment]
Had a quick look and I've made a similar change in 2b72654 which should also solve this. Would you mind trying to install the plugin from `master` by using the following command and letting me know if this also solves the problem you're running into?

```
$ npm install cascornelissen/svg-spritemap-webpack-plugin#master
```

**Comment by cascornelissen** [comment]
ping @andrejs-uibo

**Comment by andrejs-uibo** [comment]
Sorry, have not had the time to check, but it seems like it should solves the issue.

**Comment by Veron116** [comment]
There is an error if change the style option
` error  in ./node_modules/svg-spritemap-webpack-plugin/svg4everybody-helper.js`

**Comment by cascornelissen** [comment]
@Veron116, do you have any more information? Are you talking about the latest release on NPM or the `master` branch?

**Comment by cascornelissen** [comment]
This change was included in [`3.9.0`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.9.0) 🚀
